packages/transformer/src/IModelImporter.ts

```diff
@@ -109,6 +114,12 @@ export class IModelImporter {
    */
   public readonly doNotUpdateElementIds = new Set<Id64String>([]);
 
+  /** The set of elements that are added during the export of element during transformation to keep track of which elements to update during an import
+   * Defaults to an empty set.
+   * @note This is used when preserveElementIdsForFiltering is set to true
```

Comment:
I think you should mention that this is used as an optimization during the preserveElementIdsForFiltering behavior and that in normal cases where that option is false, the importer knows whether to insert or update based off of whether the ID is defined on the elementProps. The transformer already knows whether the element exists or not by time of calling importElement, so transformer pushes to the set with `markElementAsUpdate`. We can't do that in the preserveIds case because the ID should always be set whether the element exists or not. 

Maybe also mention that this set should stay small as we push to it and immediately remove from it? Or something like that?